docs/api/drivers/ResetReason.md

```diff
@@ -0,0 +1,39 @@
+## ResetReason
+
+Use the ResetReason interface to access hardware reset reason registers in a portable fashion.
```

Comment:
Could you elaborate on this? When would you want to access hardware reset reason registers in a portable fashion? Why?

Reply:
Updated.